clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/deploy/ops/DeployAppengineAtomicOperation.groovy

```diff
@@ -262,13 +262,13 @@ class DeployAppengineAtomicOperation implements AtomicOperation<DeploymentResult
       deployCommand << gcloudReleaseTrack.toString().toLowerCase()
     }
     deployCommand += ["app", "deploy", *(repositoryFullConfigFilePaths + writtenFullConfigFilePaths + configArtifactPaths)]
-    deployCommand << "--version=$versionName"
+    deployCommand << "--version=" + versionName
```

Comment:
I'd drop a comment here that `deployCommand` should be `String` only and not mixed so nobody re-learns this lesson.

Reply:
Good point, will do